src/Model/Calendar.php

```diff
@@ -100,5 +107,69 @@ public function getWeeks(): array {
 			return $weeks;
 		}
 	}
+
+	/**
+	 * Will retrieve the respective availability slots for a given calendar.
```

Comment:
Meiner Meinung nach sollte hier zur Klärung noch sowas rein: 
`Where a "slot" represents a day, regardless of the underlying internal timeframe configuration (either full-day, hourly slot-wise or custom sized slot-wise).`

Wobei ich keine Stelle gefunden habe, welche das auf technischer Ebene gut zusammenfast. Deshalb könnte der Teil in Klammern ggf. eher in die Timeframe-Klasse als hier an die Methode. Im User-Handbuch ist die Konfiguration ja auch auf zwei Einstellungen verteilt bzw. dort separat erklärt.

Reply:
Hmm, ich glaube so ganz stimmt das aber nicht. Schau dir mal den Test
testGetAvailabilitySlotsWithHourlyTimeframe, da wird ein Slot für jede Stunde des Timeframes erstellt.

Reply:
Stimmt!